api_app/analyzers_manager/file_analyzers/malprob.py

```diff
@@ -0,0 +1,79 @@
+import logging
+
+import requests
+
+from api_app.analyzers_manager.classes import FileAnalyzer
+from api_app.analyzers_manager.exceptions import AnalyzerRunException
+from tests.mock_utils import MockUpResponse, if_mock_connections, patch
+
+logger = logging.getLogger(__name__)
+
+
+class MalprobScan(FileAnalyzer):
+    url: str = "https://malprob.io/api"
+    private: bool = False
+    timeout: int = 10
```

Comment:
I got the recent time you had, I did not want to reduce this too much, we can enable a higher limit. That 60 was ok. I was curious mostly about how the service reacts

Reply:
cool!